src/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReferenceConverter.cs

```diff
@@ -12,6 +12,9 @@
 using System.Runtime.Serialization.Formatters;
 using System.Security.Permissions;
 
+// This converter is excluded from the project because its purpose is to enable displaying the properties
+// of an object implementing IComponent in the designer's Properties window and it thus has little value in corefx.
```

Comment:
instead of "little value" maybe: ... in the designer's Properties window and is thus not applicable to .NET Core